### PR TITLE
Fix batch range line selector and also allow to create logs on draft commits

### PR DESCRIPTION
**Pull request by andresgutgon**
# What?
Line picker selector was not working properly
Issue: https://github.com/latitude-dev/latitude-llm/issues/180

### 🐛 Evaluation logs not created on DRAFT commits
While fixing the batch line range selector we saw that evaluation logs where not created when in a draft commit. The reason is that we were enqueueing jobs with `document.commitId` instead of draft commit id. Now we pass `commitUuid` that is in the URL of the draft commit.

## What?
- [x] Fix draft commits not creating evaluation logs
- [x] Fix batch range line selector

### CODE REVIEW

File: packages/env/src/index.ts [hunk 1]
```diff
@@ -37,7 +37,7 @@ if (environment !== 'production') {
       WEBSOCKET_REFRESH_SECRET_TOKEN_KEY: 'refresh-refresh-token-key',
       WORKERS_WEBSOCKET_SECRET_TOKEN: 'workers-secret-token',
       DRIVE_DISK: 'local',
-      DEFAULT_PROVIDER_ID: '1',
+      DEFAULT_PROVIDER_ID: '10000000000000000000', // Crazy number to avoid conflicts
```
Comment (csansoon) [inline packages/env/src/index.ts:40]: What about -1?

Comment (andresgutgon) [reply packages/env/src/index.ts:40]: Not sure if it's a good idea to pass to postgresql a -1

**Review by csansoon — DISMISSED** [review]

File: apps/web/src/components/EditorHeader/index.tsx [hunk 2]
```diff
@@ -204,13 +206,16 @@ export default function EditorHeader({
         <Select
           name='provider'
           label='Provider'
-          placeholder='Select a provider'
+          placeholder='Using default provider'
           options={providerOptions}
           value={selectedProvider}
+          disabled={
+            disabledMetadataSelectors || isLoading || !providerOptions.length
+          }
```
Comment (geclos) [inline apps/web/src/components/EditorHeader/index.tsx:214]: does the placeholder show when disabled? we should tell the user they are using the default provider if none selected

File: packages/jobs/src/job-definitions/batchEvaluations/runBatchEvaluationJob.ts [hunk 3]
```diff
@@ -43,7 +47,7 @@ export const runBatchEvaluationJob = async (
   if (!workspace) throw new NotFoundError('Workspace not found')
 
   const commit = await new CommitsRepository(workspace.id)
-    .find(document.commitId)
```
Comment (geclos) [inline packages/jobs/src/job-definitions/batchEvaluations/runBatchEvaluationJob.ts:46]: I don't get this, how can document.commitid be different than commitUuid? the documentversion always poins to its commit no?

**Review by geclos — APPROVED** [review]

File: apps/web/src/components/EditorHeader/index.tsx (hunk 2 above)
Comment (andresgutgon) [reply apps/web/src/components/EditorHeader/index.tsx:214]: If non selected it says "default provider" yes

File: packages/jobs/src/job-definitions/batchEvaluations/runBatchEvaluationJob.ts (hunk 3 above)
Comment (andresgutgon) [reply packages/jobs/src/job-definitions/batchEvaluations/runBatchEvaluationJob.ts:46]: Before it was the commitId the document was created (live). Now is the commit that the user is seeing. live or a draft commit. So, the logs are attached to that commit. Which is what we want

**Pull request merged**